addons/edi/views/edi_gateway_views.xml

```diff
@@ -12,17 +12,20 @@
 	    <field name="can_initiate" invisible="1"/>
 	    <button name="action_transfer" type="object"
 		    string="Transfer" class="oe_highlight"
+		    groups="edi.group_edi_transfer_create"
 		    attrs="{'invisible':[('can_initiate','=',False)]}"/>
 	    <button name="action_test" type="object"
 		    string="Test Connection"
```

Comment:
Q: Does this error if the user doesn't have transfer create? I get that it is quite a bit different to the others. :)

Reply:
It was giving error for Support User because was trying to update gateway i think. So thats why added the sudo if hasrights to create, but for visibility let even for users that just want to test which can be not users that have rights to transfer.